### PR TITLE
Build on Jenkins with Gradle.

### PR DESCRIPTION
**Pull request by jeschkies**
JIRA issue: DCOS_OSS-4889

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", "usi-jenkins", ] as String[]
+
+ansiColor('xterm') {
+  // using shakedown node because it's a lightweight alpine docker image instead of full VM
+  node('shakedown') {
+    stage("Verify author") {
+      user_is_authorized(master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#marathon-dev')
```
Comment (alenkacz) [inline Jenkinsfile]: I think #marathon-dev does not exist anymore. Do we have the same thing in our jenkins files?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+    }
+  }
+  node('mesos-med') {
```
Comment (alenkacz) [inline Jenkinsfile:14]: what kind of image/node is this? I don't see it here https://jenkins.mesosphere.com/service/jenkins/computer/

File: Jenkinsfile [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+    stage('Build') {
+      try {
+        checkout scm
+        if (isUnix()) {
+          sh './gradlew clean check'
+        } else {
+          bat 'gradlew.bat clean check'
+        }
+      } finally {
+        junit(allowEmptyResults: true, testResults: 'build/test-results/test/*.xml')
```
Comment (alenkacz) [inline Jenkinsfile:24]: I am not 100% sure this is the right location but if it's working then 👍

**Review by alenkacz — APPROVED** [review]
Overall I don't have any blockers :) I was just confused with few things as mentioned in my comments.

Building jenkins plugin on jenkins :D nice 👏

File: Jenkinsfile (hunk 1 above)
Comment (jeschkies) [reply Jenkinsfile]: Good catch. I'll go with orchestration

File: Jenkinsfile (hunk 2 above)
Comment (jeschkies) [reply Jenkinsfile:14]: It's the Mesos Medium for Docker in Docker

File: Jenkinsfile (hunk 3 above)
Comment (jeschkies) [reply Jenkinsfile:24]: I tested it and it works.

**Pull request merged**